### ISSUE TITLE
result allocation: allocating too much (matrix) for points?

### ISSUE THREAD

**Issue by kristianlm** [AlexCharlton/gl-math#4]
Hi Alex!

Why is it that when I do 
```
$ csi -R gl-math -p '(v+ (make-point 1 1 1) (make-point 1 1 1))'
#f32(2.0 2.0 2.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0)
```

I get a f32vector that is large enough to hold a matrix. Is this on purpose?

**Comment by AlexCharlton** [comment]
Nope, that's a bug in the bind-math-fun macro. Thanks for finding that.
I'll patch it shortly :)

**Comment by AlexCharlton** [comment]
Fixed in d6a00be3846bbdbc24283fb2dbeed0b9719e3909

Thanks again, Kristian!
